src/dal/Jalasoft.TeamUp.Projects.DAL/ProjectsRepository.cs

```diff
@@ -64,6 +64,12 @@ public Project GetProject(Guid id)
             return result;
         }
 
+        public Project PostProject(Project project)
+        {
+            Projects.Append(project);
```

Comment:
does this work if the Projects field is readonly (line 11) ?

Reply:
sorry, my bad